cmd/skywire-cli/commands/config/gen.go

```diff
@@ -40,79 +40,170 @@ var (
 	selectedOS         string
 	disableApps        string
 	bestProtocol       bool
+	serviceConfURL     string
+	services           visorconfig.Services
+	force              bool
+	print              string
+	hide               bool
+	outunset           bool
+	svcconf            = strings.ReplaceAll(serviceconfaddr, "http://", "") //skyenv.DefaultServiceConfAddr
 )
 
+const serviceconfaddr = "http://skywire.skycoin.com"
```

Comment:
I need to update it. It should be updated in the skywire-utilities repo actually and the constant used from there.

the two vars are because the protocol doesn't need to show up in the help text, nor does the user need to specify a protocol to use a different conf service; just the (sub)domain will do